### DISCUSSION TITLE
How to set the default theme in startup

### DISCUSSION THREAD

**Discussion by satyajit-behera** [microsoft/fluentui-blazor#3037] [Q&A]
Hi, How to set the default theme to something else other than the default theme at startup.

**Comment by vnbaaij** [comment]
What other theme do you mean? Like light or dark?

**Reply by satyajit-behera** [reply]
Those color themes with application names as in MS Office

**Reply by vnbaaij** [reply]
There are examples of that in the demo site code which is part of this repo. See the `examples` folder.

Also, see https://www.fluentui-blazor.net/DesignTheme#example

**Reply by satyajit-behera** [reply]
Thanks
